[Docs site] Learning paths, v2 (#7612)
cloudflare/cloudflare-docs: content/load-balancing/_partials/_health-check-definition.md

---
_build:
  publishResources: false
  render: never
  list: never
---

Each health check is trying to answer two questions:

1. **Is the server offline?**: Does the server respond to the health check at all? If so, does it respond quickly enough (as specified in the monitor's **Timeout** field)?
2. **Is the server working as expected?**: Does the server respond with the expected HTTP response codes? Does it include specific information in the response body?

> If the answer to either of these questions result is a "No", then the server fails the health check.

Reviewer: ```suggestion
If the answer to either of these questions result in a "No", then the server fails the health check.
```

> If the answer to either of these questions result in a "No", then the server fails the health check.

Reviewer 2: ```suggestion
If the answer to either of these questions is "No", then the server fails the health check.
```
